nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/OAuth2Object.java

```diff
@@ -496,7 +581,7 @@ protected void updateTokenDoc(NuxeoOAuth2Token token, DocumentModel entry) {
     protected void deleteToken(DocumentModel token) {
```

Comment:
This could probably be moved to `OAuth2TokenService`.

Reply:
as discuss i will create a new jira to improve the whole OAuth2Object ; like remove unused params in methods, move the business code to service, avoid the use constants in interface....